### DISCUSSION TITLE
Synolocy Docker Error: SSL connect attempt failed

### DISCUSSION THREAD

**Discussion by KobayashiNori** [Difegue/LANraragi#1048] [Q&A] [answered]
![화면 캡처 2024-07-25 144006](https://github.com/user-attachments/assets/16e8bf5a-12fe-4f84-9856-554f098ae315)

![image](https://github.com/user-attachments/assets/c238ab6e-5382-456f-9ef3-e10ae839995d)

I installed lanraragi in synology docker as lastest.

But when I try to get metadata from hitomi.la, I get an error like a picture.

I just can't figure out what went wrong...

This is the content of the error.

Error while fetching tags :
Error: SSL connect attempt failed at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/Hitomi.pm line 119.

Processed 1 out of 1
Started Batch Operation...
************
Error while processing ID 924147ca696ed76d56b1cd9d20720645bfa81e85 (SSL connect attempt failed
 at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/Hitomi.pm line 119.
)

************
(code 1000)

Please Help me ..!

**Comment by yes89929** [comment]
same issue.
In my case, "hentag online lookups" works, but "e-hentai" and "hitomi" give an ssl error.
My configuration is synology.

**Comment by KobayashiNori** [comment, marked as answer]
> same issue. In my case, "hentag online lookups" works, but "e-hentai" and "hitomi" give an ssl error. My configuration is synology.

I solved.
It's been a government censorship issue. It's solved by using OPEN VPN for containers or NAS.
